### DISCUSSION TITLE
Upgrade Guide from 3.1.x to 3.4.x

### DISCUSSION THREAD

**Discussion by arunim2405** [GeekyAnts/NativeBase#5347] [General]
### Description

As there were multiple changes from 3.1.x to 3.2.x and then again to 3.4.x, an all in one guide for upgrading to 3.4.x would be nice

**Comment by Viraj-10** [comment]
Hi @arunim2405, Thanks for the query. If you could provide some details about what difficulties you are facing while upgrading from 3.1.x to 3.4.x. We can help you resolve those issues.  We always make our best efforts to avoid breaking changes. Meanwhile, we will try to provide a combined guide ASAP.
